client/my-sites/plan-features/index.jsx

```diff
@@ -165,6 +169,37 @@ class PlanFeatures extends Component {
 		);
 	}
 
+	render30PercentOffNotice() {
+		const {
+			canPurchase,
+			hasPlaceholders,
+			translate,
+			withSaleInfo,
+			isEligibleForSpringDiscount: eligible,
+		} = this.props;
+		const bannerContainer = document.querySelector( '.plans-features-main__notice' );
+
+		if ( ! bannerContainer ) {
+			return false;
+		}
+
+		if ( ! eligible || ! withSaleInfo || hasPlaceholders || ! canPurchase ) {
+			return ReactDOM.createPortal( <div />, bannerContainer );
+		}
+
+		return ReactDOM.createPortal(
+			<Notice
+				className="plan-features__notice-credits"
+				showDismiss={ false }
+				icon="info-outline"
+				status="is-success"
+			>
+				{ translate( 'Enter coupon code “SPRING30” during checkout to claim your 30% discount' ) }
```

Comment:
Could you please convert the coupon code and the discount percentage to variables so that we'll be able to reuse translations and also not confuse translators around the translation of "SPRING30." Thanks!

Reply:
Thanks! Actually I realized I could remove `translate()` altogether.

Reply:
No need for the comment actually. I think our parser is intelligent enough to ignore the verbatim `translate()` but still we don't need to test it :)